### PR TITLE
V0.1.5

### PR DESCRIPTION
**Pull request by reiniervlinschoten**
* Updated readme and setup
* Fixed tests for 2022.3 release and made error tests more robust
* Added correct StudyIDs for testing on Github
* Added approporiate user-agent header
* Updated requirements

### CODE REVIEW

**Comment by reiniervlinschoten** [comment]
@zoni , I have added your changes here, tell me if it is okay.

File: castoredc_api/client/castoredc_api_client.py [hunk 1]
```diff
@@ -41,18 +44,29 @@ def __init__(self, client_id, client_secret, url):
         self.base_url = f"https://{url}/api"
         self.auth_url = f"https://{url}/oauth/token"
 
+        try:
+            self.package_version = pkg_metadata.version("castoredc_api")
+        except pkg_metadata.PackageNotFoundError:
+            self.package_version = "Github Tests"
```
Comment (zoni) [inline castoredc_api/client/castoredc_api_client.py:50]: I would probably use:

```suggestion
        try:
            self.package_version = pkg_metadata.version("castoredc_api")
        except pkg_metadata.PackageNotFoundError:
            self.package_version = "unknown"
```

Or even:

```suggestion
        try:
            self.package_version = pkg_metadata.version("castoredc_api")
        except pkg_metadata.PackageNotFoundError:
            self.package_version = "dev"
```

File: castoredc_api/tests/test_api_endpoints/test_country_endpoints.py [hunk 2]
```diff
@@ -86,22 +86,22 @@ def test_single_country_failure_too_large(self, client):
         """Tests if retrieving a non-existent (edge case: upper range) country ID raises an error."""
         with pytest.raises(HTTPStatusError) as e:
             client.single_country(252)
-        assert "404 Client Error: Not Found for url" in str(e.value)
+        assert "Client error '404 Not Found" in str(e.value)
```
Comment (zoni) [inline castoredc_api/tests/test_api_endpoints/test_country_endpoints.py:89]: This test, and the ones below and in other testcases, would probably be less brittle if you check the status code as opposed to the string value. Off the top of my head, I believe the attribute is `response.status_code`:

```suggestion
        assert e.response.status_code == 404
```

**Review by zoni — DISMISSED** [review]
I cannot comment on the API changes as, even though I'm employed by Castor, I don't really work in that area. But the overall changes look good to me!

Just have minor suggestions for a few small improvements

File: castoredc_api/client/castoredc_api_client.py (hunk 1 above)
Comment (reiniervlinschoten) [reply castoredc_api/client/castoredc_api_client.py]: Changed this to your suggestion
